doc/source/creating.rst

```diff
@@ -0,0 +1,29 @@
+***********************
+Creating a New Database
+***********************
+
+In some cases, you may want to create a new database. Currently, storage and
+retrieval is implemented in MongoDB so instructions for MongoDB will follow.
+The implementation for other types of databases is likely similar.
+
+Creation in MongoDB
+-------------------
+
+The creation of a database in MongoDB is simple. Simply create a configuration
+as described in :ref:`configuration_doc`. The only requirement is that the MongoDB
```

Comment:
If you just want to point to the configuration.rst file generically (rather than a specific section within it, you can use the directive

```
:doc:`configuration`
```

with no need to create a specific reference.

Reply:
great thanks for the advice!